spec/models/teacher_spec.rb

```diff
@@ -8,4 +8,16 @@
     it { is_expected.to validate_presence_of(:email) }
     it { is_expected.to validate_presence_of(:password) }
   end
+
+  describe 'email validation fail' do
+    it 'for unique email' do
+      teacher_1 = create(:teacher, attributes_for(:teacher))
+      teacher_2 = build(:teacher, attributes_for(:teacher, email: teacher_1.email))
+      expect(teacher_2).not_to be_valid
+    end
+  end
+
+  describe 'validation fail due to password too short' do
+    let(:teacher) { Teacher.new(attributes_for(:teacher, password: 'pea')) }
```

Comment:
We're missing the actual test here - there's only `let`, no `it` no `expect` ;)

Reply:
Check :)